.github/workflows/docs_manual.yaml

```diff
@@ -19,19 +19,19 @@ jobs:
       url: ${{ steps.deployment.outputs.page_url }}
     runs-on: ubuntu-22.04
     steps:
-      - uses: actions/checkout@v4
-      - uses: actions/setup-node@v4
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
```

Comment:
While I understand this does reduce security risks, is it worth the extra effort, especially for such well-known actions?
I find it... curious... that on their [own website](https://github.com/actions/dependency-review-action) their examples do NOT follow this guideline:
```
  dependency-review:
    runs-on: ubuntu-latest
    steps:
      - name: 'Checkout Repository'
        uses: actions/checkout@v4
      - name: Dependency Review
        uses: actions/dependency-review-action@v4
    . . .
```

Reply:
There's little extra effort after the initial setup: dependabot understands this and updates the refs.